exchange/utils.go

```diff
@@ -45,7 +45,16 @@ func cleanOpenRTBRequests(ctx context.Context,
 		for bidder, bidReq := range requestsByBidder {
 			// Fixes #820
 			coreBidder := resolveBidder(bidder.String(), aliases)
-			if ok, err := gDPR.PersonalInfoAllowed(ctx, coreBidder, consent); !ok && err == nil {
+
+			var publisher_id string
+			if bidReq.Site != nil && bidReq.Site.ID != "" {
+				publisher_id = bidReq.Site.ID
+			} else if bidReq.App != nil {
+				publisher_id = bidReq.App.ID
```

Comment:
Likewise It is `bidReq.App.Publisher.Id`, we are identifying the publisher of the app.